upgrading/extending-daml-models/1_package_ids.md

```diff
@@ -0,0 +1,89 @@
+You've already learned how to extend a DAML model when you added the messaging feature to the mini
+social network of the [Getting Started
+Guide](https://daml.com/learn/getting-started/your-first-feature). 
+
+There's a catch though with extending a DAML model like this. The package ID of a DAML archive
+depends on the contents of the compiled DAML code. Changing the code results in a different package
```

Comment:
Maybe briefly explain how packages work in DAML/what they are

Reply:
done.